_data/retail.yml

```
@@ -262,11 +304,16 @@ websites:
     img: lowes.png
     twitter: Lowes
     facebook: lowes
+    regions:
+      - ca
+      - us
 
   - name: Macy's
     url: https://www.macys.com/
     twitter: Macys
     img: macys.png
+    regions:
+      - us
```

```suggestion
    img: macys.png
```
Macy's [ships internationally](https://www.macys.com/international/context?cm_sp=navigation-_-bottom_nav-_-change_country&lid=glbbtmnav_change_country-us).